.github/workflows/release-python-client.yaml

```diff
@@ -0,0 +1,55 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-18.04
```

Comment:
What is your thought behind pinning this version? In our other actions we set this to "ubuntu-latest."